crates/bevy_render/src/view/visibility/render_layers.rs

```diff
@@ -492,6 +492,21 @@ mod rendering_mask_tests {
             2,
             "layer 0 + 1 - 0 is mask 2"
         );
+        assert_eq!(
+            RenderLayers::from_iter((0..=63).into_iter().map(|_| 0))
+                .add(1)
+                .layers[0],
+            3,
+            "layer (64 * 0) + 1 is mask 3"
```

Comment:
Perhaps this wording is a bit confusing, but I'm not sure how to put it concisely.